Salesforce Org Setup/force-app/main/default/lwc/incidentMap/incidentMap.js

```diff
@@ -0,0 +1,44 @@
+import {
+  LightningElement,
+  wire,
+  api,
+  track
+} from 'lwc';
+import {
+  getRecord
+} from 'lightning/uiRecordApi';
+
+const FIELDS = [
```

Comment:
Make this schema imports.

Reply:
Couldn't get that working with geolocation fields. This, however, does.